articles/azure-fluid-relay/how-tos/container-recovery.md

```diff
@@ -12,55 +12,69 @@ In this scenario, we explore data recovery. We consider data to be corrupted whe
 
 ## How Fluid Framework and Azure Fluid Relay save state
 
-Fluid framework periodically saves state, called summary, without any explicit backup action initiated by the user. This workflow occurs every one (1) minute if there's no user activity, or sooner if there are more than 1000 pending ops present. Each pending op roughly translates to an individual user action (select, text input etc.) that wasn't summarized yet.
+Fluid Framework periodically saves snapshots of the data in the container, which summarize all changes made to the data up to that point.  During normal loading the latest snapshot is retrieved, and any subsequent changes are applied on top of that state.
+
+If the latest snapshot or subsequent changes are corrupt, Fluid may not be able to load them normally.  In this case, Fluid offers a collection of APIs to view the stored snapshot versions and load them in a view-only mode with no subsequent changes applied.  This allows the data to be extracted and optionally injected into a new container to resume collaboration.
 
 ## Azure client APIs
 
-We added the following methods to AzureClient that enable developers to recover data from corrupted containers. 
+## APIs for viewing and loading container versions
```

Comment:
Can't have 2 H2 in a row like this. There must be some content beneath every H2.